rfcs/0003-standard-gh-labels.md

```diff
@@ -0,0 +1,110 @@
+
+See https://github.com/cds-snc/covid-alert-server-labels for an example of what is being proposed to implement this solution. 
+
+The source of truth for these labels will reside in github and what is put forth in this document is just to start the discussion. The understanding is that if this Standard is accepted that a github action for syncing labels in a project will be created that will be created in github and maintenance and discussion will continue in those repositories. 
+
+### Baseline Labels
+This list of labels will be standardized across the organization and will be mandatory, they will apply high level groupings and statuses and will allow for a consistent way of talking about issues across projects.
+
+This consistent vocabulary will enable us to create reporting tools that can be applied to any project using the system and will provide a consistent view into the state of all CDS products. It should also reduce cognitive burden when onboarding to the project. 
+
+The list of proposed baseline labels can be seen in Appendix A
+
+### Product Specific Labels
+
+Each product will be allowed to apply their own unique set of labels alongside the baseline labels. 
+
+
+```yaml
+- Name: High Priority | Haute priorité
+  Description: 
+  Colour: 
+- Name: Medium Priority  | Priorité moyenne
+  Description:
+  Colour:
+- Name: Low Priority | Faible priorité
+  Description:
+  Colour
+- Name: Bug | Bogue
+  Description:
+  Colour:
+- Name: Documentation
+  Description:
+  Colour:
+- Name: Security | Sécurité
+  Description:
+  Colour:
+- Name: Privacy | Vie privée
+  Description:
+  Colour:
+- Name: Wont fix | Ne résoudra pas
+  Description:
+  Colour:
+```
+
+
+### Label Requirements
+
+Every label is bound by the following requirement:  
+
+- A label must be bilingual in the format `en | fr`
+- A label must not use a colour used by a baseline label
+
+It is recommended that labels make sense for the product/project they are being applied to and that they are not relevant to only a very small number of issues but this isn’t mandatory.
+
+### Governance
+
+Governance surrounding labels will occur in Github repos, as labels are defined as json objects and are enforced through Github actions we can use Github to apply a level of lightweight automated workflows. Github's Code owner functionality will be used to manage approvals from owners.
+
+#### Creation, Modification, and Approval
+Label Meta-data is defined by Github and can be found here https://docs.github.com/en/rest/reference/issues#create-a-label
+
+The governance surrounding the creation, modification and approval of labels will depend on what label group is being acted upon. 
+
+**CDS wide labels** will be owned by the head of product management. 
```

Comment:
too soon for this proposed line, I suggest we don't dive into ownership outside dev co

Reply:
Makes sense will update

Reply:
Changing to head of development.